linux/keyman-config/debian/control

```diff
@@ -3,8 +3,7 @@ Maintainer: Daniel Glassey <wdg@debian.org>
 Section: python
 Priority: optional
 Build-Depends: dh-python, python3-setuptools, python3-all, debhelper (>= 9), bash-completion,
- python3-magic, python3-qrcode <!pkg.keyman-config.xenial>,
- python-qrcode <pkg.keyman-config.xenial>
+ python3-magic, python3-qrcode
```

Comment:
I think this breaks package build on Xenial because python3-qrcode isn't available. We might have to use two different `debian/control` files for Xenial and >= Bionic (e.g. create a template file and then pre-process that to create the desired `debian/control` file before creating the source package)

Reply:
I'll revert those changes